src/main/routes/routes.ts

```diff
@@ -9,6 +11,8 @@ export default function(app: Application): void {
   app.post('/lip-or-representative', app.locals.container.cradle.lipOrRepController.post);
   app.get('/single-or-multiple-claim', app.locals.container.cradle.singleOrMultipleController.get);
   app.post('/single-or-multiple-claim', app.locals.container.cradle.singleOrMultipleController.post);
+  app.get('/multiple-respondent-check', new MultipleRespondentCheckController(multipleRespondentContent).get);
```

Comment:
You shouldn't need to initialise the class like that anymore. The configuration is setup so all the links should work the same way now.

Reply:
amended accordingly